src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -35,7 +36,11 @@ ContinuousBatchingPipeline::ContinuousBatchingPipeline( const std::filesystem::p
                                                         const ov::AnyMap& tokenizer_properties) {
     auto properties_without_draft_model = properties;
     auto draft_model = extract_draft_model_from_config(properties_without_draft_model);
```

Comment:
what if both draft model and prompt lookup are specified? 

Looks like we need some validation specified properties.

Reply:
done